packages/ppx/browser.ml

```diff
@@ -36,6 +36,11 @@ module Effect = struct
 end
 
 module Browser_only = struct
+  let impossible ~label loc message =
+    let message = "[" ^ label ^ "]" ^ ": " ^ message in
```

Comment:
Maybe instead of the label we should render the location directly?

Reply:
I'm afraid that those labels are very Parsetree specific and not user-centric

Reply:
Good idea. Using the location could also reduce code size by avoiding copying large strings of the function definition.